spec/workers/delivery_request_worker_spec.rb

```diff
@@ -28,63 +27,39 @@
         described_class.new.perform(
           email.id,
           { "content_change_created_at" => Time.zone.now.iso8601 },
+          queue,
         )
       end
     end
 
     it "increments the rate limiter" do
       expect(rate_limiter).to receive(:add).with("delivery_request")
-      described_class.new.perform(email.id)
+      described_class.new.perform(email.id, {}, queue)
     end
 
     context "when rate limit is exceeded" do
-      it "raises a RatelimitExceededError" do
-        allow(rate_limiter).to receive(:exceeded?).and_return(true)
-        expect { described_class.new.perform(email.id) }
-          .to raise_error(described_class::RateLimitExceededError)
+      around { |example| Sidekiq::Testing.fake! { example.run } }
```

Comment:
I don't think so as it would then affect other tests. In order to do that we'd need a different before block that always set it to inline, whereas the code for this app just sets to inline before all tests start: https://github.com/alphagov/email-alert-api/blob/aed8bfa357c7e415007e85be3a3e811f8b9c0e3c/spec/spec_helper.rb#L38

Reply:
Oh okay I see why, nm.

I think I've also noticed a duplicate line [here](https://github.com/alphagov/email-alert-api/pull/1341/files/10b5f4eea4ad7f45e17d7e27793d15eaf099a4f5..f1348ae768294568c39385cacdab78f0b1e8a6d0#diff-ba6f7abd0285f948da91a0be14d695e2R46) by the way that's already specified in the before block.

Reply:
Oh thanks I'll sort that.